### PR TITLE
Easyconfig files adapted to use the new MPICH easyblocks

### PR DESCRIPTION
**Pull request by damianam**
See https://github.com/hpcugent/easybuild-easyconfigs/issues/2334 and https://github.com/hpcugent/easybuild-easyblocks/pull/818

### CODE REVIEW

**Comment by damianam** [comment]
ping

File: easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'EB_MPICH2'
+
+name = 'psmpi'
+version = '5.1.5-1'
+
+homepage = 'https://github.com/ParaStation/psmpi2'
+description = """ParaStation MPI as part of the ParaStationV5 cluster suite provides robust, flexible and scalable communication and management functions for Linux-based compute clusters. Beside parallel applications based on the Message Passing Interface specification, version 2 (MPI2), also serial applications are supported.
```
Comment (boegel) [inline easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb]: tooooooo long :)

File: easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2b-GCC-4.9.3.eb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = "This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH."
+
+toolchain = {'name': 'GCC', 'version': '4.9.3'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/']
+sources = [SOURCELOWER_TAR_GZ]
+
+# Let's store the checksum in order to be sure it doesn't suddenly change
+#checksums = ['77bba29442f81e196155e87869cdceda']
```
Comment (boegel) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2b-GCC-4.9.3.eb]: either use the checksums, or don't

File: easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+dependencies = [
+    ('pscom', '5.0.45-1', '', True),
+]
+
+hiddendependencies = [
+    ('pscom', '5.0.45-1', '', True),
+]
+
+# Build shared libraries
+configopts = ' --with-confset=gcc PSCOM_LDFLAGS=-L$EBROOTPSCOM/lib PSCOM_CPPFLAGS=-I$EBROOTPSCOM/include --with-mpichconf="--enable-static" '
```
Comment (boegel) [inline easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb]: too long?

File: easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+modluafooter = 'family("mpi")'
```
Comment (boegel) [inline easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-GCC-4.9.3.eb]: hmm, site-specific?

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@damianam: do we want to update existing MPICH and MVAPICH2 easyconfigs too, to use the (changed) easyblocks in https://github.com/hpcugent/easybuild-easyblocks/pull/818 ?

If so, let's do that in a separate PR...

**Comment by damianam** [comment]
@boegel you have a new PR for the updated easyconfigs: https://github.com/hpcugent/easybuild-easyconfigs/pull/2505

**Comment by damianam** [comment]
@boegel I am closing this PR and I'll make a new one with the changes needed for https://github.com/hpcugent/easybuild-easyblocks/pull/844

**Comment by boegel** [comment]
@damianam OK, thanks

Just for reference: see also #2589

**Pull request closed without merging**